sunkit_image/granule.py

```diff
@@ -166,44 +180,47 @@ def _mark_brightpoint(segmented_image, data, resolution, bp_min_flux=None):
     Returns
     -------
     segmented_image_fixed : `numpy.ndrray`
-        The segmented image with brightpoints marked as 3.
+        The segmented image with brightpoints marked as 2.
     brightpoint_count: `int`
         The number of brightpoints identified in the image.
     granule_count: `int`
         The number of granules identified, after re-classifcation of brightpoint.
     """
+    # General size limits 
     bp_size_limit = (
         0.1  # Approximate max size of a photosphere bright point in square arcsec (see doi 10.3847/1538-4357/aab150)
     )
-    bp_pix_upper_limit = bp_size_limit / (resolution**2)
+    bp_pix_upper_limit = (bp_size_limit / resolution)**2 # Max area in pixels
     bp_pix_lower_limit = 4  # Very small bright regions are likely artifacts
-    # General flux limit determined by visual inspection.
+    # General flux limit determined by visual inspection (set using equalized map)
     if bp_min_flux is None:
-        stand_devs = 0.5
-        bp_brightness_limit = np.mean(data) + stand_devs * np.std(data)
+        stand_devs = 1.25

```

Comment:
What was the reason to increase this?

Reply:
The minimum flux is now based on the histogram equalized image, and I found that 1.25*SD was what worked most consistantly on those images.

Reply:
Can you leave a comment as to that here jus for future reference please?